pallets/messages/src/weights.rs

```diff
@@ -56,78 +56,78 @@ pub trait WeightInfo {
-	/// Storage: Messages Messages (r:1 w:1)
-	/// Proof: Messages Messages (max_values: None, max_size: Some(771174), added: 773649, mode: Measured)
-	/// The range of component `n` is `[0, 5119]`.
+	/// Storage: `Schemas::Schemas` (r:1 w:0)
+	/// Proof: `Schemas::Schemas` (`max_values`: None, `max_size`: Some(65518), added: 67993, mode: `MaxEncodedLen`)
+	/// Storage: `Msa::PublicKeyToMsaId` (r:1 w:0)
+	/// Proof: `Msa::PublicKeyToMsaId` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
+	/// Storage: `Msa::DelegatorAndProviderToDelegation` (r:1 w:0)
+	/// Proof: `Msa::DelegatorAndProviderToDelegation` (`max_values`: None, `max_size`: Some(217), added: 2692, mode: `MaxEncodedLen`)
+	/// Storage: `Messages::MessagesV2` (r:0 w:1)
+	/// Proof: `Messages::MessagesV2` (`max_values`: None, `max_size`: Some(3125), added: 5600, mode: `MaxEncodedLen`)
+	/// The range of component `n` is `[0, 3071]`.
 	fn add_onchain_message(n: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `7339`
-		//  Estimated: `19714`
-		// Minimum execution time: 39_000_000 picoseconds.
-		Weight::from_parts(179_112_822, 19714)
-			// Standard Error: 35
-			.saturating_add(Weight::from_parts(459, 0).saturating_mul(n.into()))
-			.saturating_add(T::DbWeight::get().reads(4_u64))
+		//  Measured:  `443`
+		//  Estimated: `77893`
+		// Minimum execution time: 24_000_000 picoseconds.
+		Weight::from_parts(24_385_954, 77893)
+			// Standard Error: 43
+			.saturating_add(Weight::from_parts(497, 0).saturating_mul(n.into()))
+			.saturating_add(T::DbWeight::get().reads(3_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
-	/// Storage: Schemas Schemas (r:1 w:0)
-	/// Proof: Schemas Schemas (max_values: None, max_size: Some(65518), added: 67993, mode: Measured)
-	/// Storage: Msa PublicKeyToMsaId (r:1 w:0)
-	/// Proof: Msa PublicKeyToMsaId (max_values: None, max_size: Some(48), added: 2523, mode: Measured)
-	/// Storage: Messages Messages (r:1 w:1)
-	/// Proof: Messages Messages (max_values: None, max_size: Some(771174), added: 773649, mode: Measured)
+	/// Storage: `Schemas::Schemas` (r:1 w:0)
+	/// Proof: `Schemas::Schemas` (`max_values`: None, `max_size`: Some(65518), added: 67993, mode: `MaxEncodedLen`)
+	/// Storage: `Msa::PublicKeyToMsaId` (r:1 w:0)
+	/// Proof: `Msa::PublicKeyToMsaId` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
+	/// Storage: `Messages::MessagesV2` (r:0 w:1)
+	/// Proof: `Messages::MessagesV2` (`max_values`: None, `max_size`: Some(3125), added: 5600, mode: `MaxEncodedLen`)
 	fn add_ipfs_message() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `6183`
-		//  Estimated: `18558`
-		// Minimum execution time: 36_000_000 picoseconds.
-		Weight::from_parts(174_485_000, 18558)
-			.saturating_add(T::DbWeight::get().reads(3_u64))
+		//  Measured:  `915`
+		//  Estimated: `77893`
```

Comment:
what is making the estimates so far off?

Reply:
It's because of the mode of benchmark changed from `Measured` to `MaxEncodedLen`. This increase is temporary since the PR for Schema changes is not merged we are not seeing the effect of that here yet.